docs/tutorials/t2ramsey_tutorial.ipynb

```diff
@@ -0,0 +1,416 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# T_2 Ramsey Experiment"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This experiment serves as one of the series of experiments used to characterize a single qubit. Its purpose is to determine two of the qubit's properties: *Ramsey* or *detuning frequency* and $T_2\\ast$. The rough frequency of the qubit was already determined previously. Here, we would like to measure the *detuning*, that is  the difference between the qubit's precise frequency and the frequency of the rotation pulses (based on the rough frequency). This part of the experiment is called a *Ramsey Experiment*. $T_2\\ast$ represents the rate of decay toward a mixed state, when the qubit is initialized to the |+⟩ state."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "import qiskit\n",
+    "from qiskit_experiments.characterization import T2Ramsey"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The circuit used for the experiment comprises the following:\n",
+    "\n",
+    "    1. Hadamard gate\n",
+    "    2. delay\n",
+    "    3. p gate that rotates the qubit in the x-y plane \n",
+    "    4. Hadamard gate\n",
+    "    5. measurement\n",
+    "\n",
+    "During the delay time, we expect the qubit to precess about the z-axis. If the p gate and the precession offset each other perfectly, then the qubit will arrive at the |1⟩ state (after the second Hadamard gate). By varying the extension of the delays, we get a series of oscillations of the qubit state between the |0⟩ and |1⟩ states. We can draw the graph of the resulting function, and can analytically extract the desired values."
```

Comment:
> then the qubit will arrive at the |1⟩ state (after the second Hadamard gate)

Since you are not doing X90-deley-X90 but rather H-delay-H, the qubit returns to |0> (with perfect tuning), isn't it ?

Reply:
You are right - nice catch!